src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.1.0
+-----
+
+ * `ExceptionListener` now logs and collects exceptions at priority `2048` (previously logged at `-128` and collected at `0`)
```

Comment:
Is it worth to add this to the upgrade file too?

Reply:
What exactly needs an upgrade step?

Reply:
I was thinking about other listeners that relied on this one being executed before/after themselves. Probably an edge case though that's not needed to be tackled.